Node Status Monitor/NodeJS/node_monitor.js

```diff
@@ -0,0 +1,197 @@
+/* imports*/
+import fetch from 'node-fetch';
+import fs from 'fs';
+import log from './utils/logger.js';
+import { get_token } from './api/auth.js';
+import config from './config/config.js';
+
+
+/* 
```

Comment:
These variables and functions should be commented inline.

Reply:
added comment